lib/prawn-fillform.rb

```diff
@@ -297,7 +286,9 @@ def acroform_fields_for_page(page)
     end
 
     def fetch_field_attribute(data, page, field_name, attribute)
-      value = data[page][field_name].fetch(attribute) rescue nil
+      page_number = page.to_s.split('_').last.to_i + 1
+      page_key = "page_#{page_number}".to_sym
+      value = data[page_key][field_name].fetch(attribute) rescue nil
```

Comment:
It's a bug fix, not related to the check box. If I don't make this change, the example output will be broken (no image, no first name).  This fixes the behavior so that you can override attributes for a particular page.  I can remove this if you don't think it's necessary.